AprilSylph/XKit-Rewritten
# Quick Reblog: Prevent context menu interference on touchscreens

Browsers:
 - Kiwi Browser, Android: tested, **works**
 - Firefox nightly, Android: tested, **doesn't work reliably** 
 - Safari, iOS via packaging this extension with XCode: untested; probably does not work (see below)
 - Chromium, Windows w/ touchscreen: tested, **works**
 - Firefox, Windows w/ touchscreen: tested, **works**

### Description


This prevents the context menu that appears when long-pressing the reblog button using a touchscreen from popping up, allowing that gesture to be used to open the quick reblog menu.

Provides a resolution for #791 given that you aren't using Firefox for Android. (If you are, it seems to... sort of help? There's a specific duration that you can do the long press that seems to work.)

### Testing steps


- Enable Quick Reblog on a device with a touch screen.
- Long-press an active reblog button. The Quick Reblog popup should appear, and no right-click context menu should appear, if you are using a device/browser combination that this supports.
